tools/tune/eq/eq_get_abi.m

```diff
@@ -11,18 +11,55 @@
 %
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 
-%% Use sof-ctl to write ABI header into a file
+ipc4_use_sofctl = 0; % Set to 1 when sof-ctl update is merged
 abifn = 'eq_get_abi.bin';
-cmd = sprintf('sof-ctl -g %d -b -o %s > /dev/null', setsize, abifn);
-system(cmd);
+switch ipc_ver
+	case 3,
```

Comment:
In the `case` statement, an extra comma is not essential.
the same applies to other locations

Reply:
Yep, some old habit that's no more in the examples. But I think it's needed when on the same line, e.g. `case PEQ_HP1, [z0, p0, k0] = butter(1, 2*f/fs, 'high');`